addon/utils/absoluteUrl.js

```diff
@@ -1,5 +1,5 @@
 export default url => {
-  return url.indexOf("http") === 0
+  return /http(s)?/.test(url)
```

Comment:
This is just more explicit, but the behavior actually doesn't change, right?

Reply:
In this version yes, I changed the code again - sorry :wink: